go/libkb/keyfamily.go

```diff
@@ -728,6 +739,36 @@ func (ckf ComputedKeyFamily) GetActivePGPKeys(sibkey bool) (ret []*PGPKeyBundle)
 	return
 }
 
+type RevokedKey struct {
+	Key       GenericKey
+	RevokedAt *KeybaseTime
+}
+
+func (ckf ComputedKeyFamily) GetRevokedKeys() []RevokedKey {
+	var revokedKeys []RevokedKey
+	for kid := range ckf.kf.AllKIDs {
+		ki := ckf.cki.Infos[kid]
+		if ki == nil {
+			continue
```

Comment:
If these functions already had errors in their signatures, I'd be for making this condition an error. I wouldn't want to change the signature to handle a case that should never happen, though.

My instinct is to panic in these cases, but you made a good point below about how we shouldn't be killing the whole service :( Maybe a longer-term answer is to put a `recover` block around the service's main loop? In the meantime, I guess we could just throw something in the error log and call it a day?